actors/builtin/miner/miner_test.go

```diff
@@ -2785,6 +2847,17 @@ func (h *actorHarness) terminateSectors(rt *mock.Runtime, sectors bitfield.BitFi
 	if !pledgeDelta.Equals(big.Zero()) {
 		rt.ExpectSend(builtin.StoragePowerActorAddr, builtin.MethodsPower.UpdatePledgeTotal, &pledgeDelta, big.Zero(), nil, exitcode.Ok)
 	}
+	if len(dealIDs) > 0 {
```

Comment:
Nice.  How did you find out about this one?  Did the above test somehow exercise more deal ids than the cbg.MaxLength limit?  That would be surprising since I don't see any deal addition logic explicitly.

Reply:
We just didn't have any terminate sector tests that exercised sectors with deals. Having at least 1 deal is a requirement of replaced capacity sectors, so the new test broke and I had no choice but to fix this.